plugin/clang.vim

```diff
@@ -864,7 +914,24 @@ func! s:ClangExecute(root, clang_options, line, col)
   let l:tmps = [tempname(), tempname()]
   let l:command .= ' 1>'.l:tmps[0].' 2>'.l:tmps[1]
   let l:res = [[], []]
-  if !exists('v:servername') || empty(v:servername)
+  if has("nvim")
+    " Try to make the most use of Neovim job control.
+    " Neovim will not freeze when it handles large codebase projects.
+    " TODO: 
+    let s:tmps = l:tmps
+    let l:nvimsrc1 = split(l:src, "'")
```

Comment:
I see, the job start need an `argv`, which is an array. You'd better don't use `split` the nvimcmd to construct this array, this is buggy(e.g. have space in file path). You should construct `argv` manually, e.g.
```viml
let l:argv = [g:clang_exec, '-fsyntax-only', ...]
```
Can you help to test a source path contains space will work or not when the `l:src` is not quoted ?
I guess it may be OK, as the `jobstart` way is different with the `system` way.
Thank you.